lib/index.js

```diff
@@ -1,7 +1,9 @@
 "use strict";
```

Comment:
Please can you revert this file? Looks like no changes were made except formatting, and this will cause annoying conflicts if we ever need to sync the fork with the base